README.md

```diff
@@ -38,15 +43,16 @@ tags = {
 
 <!-- markdownlint-disable MD003 MD013 MD022 MD033 -->
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
What sorcery is this?!?  Is `pre-commit` autopopulating this section for you or something?

Reply:
You know, I'm not sure - it existed from before I touched the repo and remained unchanged. Mmmmmagic.

Reply:
@mcdonnnj or @felddy, do you know about this jazz?